index.bs

```diff
@@ -581,23 +592,23 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Let |clientDataHash| be the [=hash of the serialized client data=] represented by |clientDataJSON|.
 
-1. Let |currentlyAvailableAuthenticators| be a new [=ordered set=] consisting of all [=authenticators=] 
+1. Let |currentlyAvailableAuthenticators| be a new [=ordered set=] consisting of all [=authenticators=]
```

Comment:
well, various people complain about spurious whitespace lingering at end-of-lines, or when parags are reformatted (we agreed to a 128-char line length way back when). 

I note that viewing the github-generated diff using `?w=1` at end of URI, eg <https://github.com/w3c/webauthn/pull/464/files?w=1> helps filter-out whitespace-only changes.   

when using git diff locally, there's `git diff -w`